app/views/events/partials/_invitation.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if user_signed_in? %>
+  <%= render 'invitations/invitation', event: @event %>
+<% else %>
+  <h3>To attend this event you need to log in!</h3>
+<% end %>
```

Comment:
- [ ] Kindly remove logic from your views. This is important because it allows your application to follow the MVC pattern. You add all your view logic inside the helpers. Kindly follow this article to learn more about this.
[Article](http://codefol.io/posts/Where-Do-I-Put-My-Code/)

<img width="825" alt="no_logic_in_helpers_requirement" src="https://user-images.githubusercontent.com/32361963/109273044-b83a9400-7822-11eb-986b-d93639cf6e35.png">


**NB:**
 kindly note this applies to all other parts of your application. Make sure there is no logic in your views